sdk/monitor/Azure.Monitor.Ingestion/samples/Sample1_LogData.md

```diff
@@ -0,0 +1,57 @@
+# <scenario_list>
+
+To use these samples, you'll first need to set up resources. See [getting started](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/Azure.Monitor.Ingestion/README.md#getting-started) for details.
+
+## <scenario>
+
+You can create a client and call the client's `Upload` method. Please note that at this time the upload limit is 1 Mb. Any content uploaded larger than 1 Mb will throw an exception.
```

Comment:
Since the upload limit is owned by the service team and is subject to change, it may be best to link to their doc instead of hard-coding the limit here. We want their doc to be the source of truth.